apps/minifront/src/state/staking.ts

```diff
@@ -279,6 +290,50 @@ export const createStakingSlice = (): SliceCreator<StakingSlice> => (set, get) =
       });
     }
   },
+  undelegateClaim: async () => {
```

Comment:
These state methods that build and submit a transaction, as well as manage toasts, are quite repetitive. I want to DRY them up, but this PR is already pretty huge (sorry), so I'll do that in a future PR.